lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2560,6 +2560,7 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
 
   swift_ast_sp->InitializeSearchPathOptions(module_search_paths,
                                             framework_search_paths);
+  swift_ast_sp->GetCompilerInvocation().computeCXXStdlibOptions();
```

Comment:
What information does this function depend on? Would it make sense to sink this into InitializeSearchPathOptions/ConfigureResourceDirs/SetCompilerInvocationLLDBOverrides?

Reply:
Sounds good, I moved this into `InitializeSearchPathOptions`.

`computeCXXStdlibOptions` depends on the ClangImporter options and on the SDK/search path being configured.